crates/ingress-kafka/src/consumer_task.rs

```diff
@@ -161,20 +162,19 @@ impl MessageSender {
         ordering_key_format: &KafkaOrderingKeyFormat,
         ordering_key_prefix: &str,
         msg: &impl Message,
-    ) -> Bytes {
-        let mut buf = BytesMut::new();
-        buf.put(ordering_key_prefix.as_bytes());
-        buf.put(msg.topic().as_bytes());
-        buf.put_i32(msg.partition());
+    ) -> String {
+        let mut buf = String::new();
```

Comment:
Probably premature optimization: I guess we could create the String with a predetermined capacity.